kitsune/settings.py

```diff
@@ -426,6 +425,7 @@ def immutable_file_test(path, url):
 
 TEMPLATES = [
     {
+        "NAME": "jinja2",
```

Comment:
This is necessary for our custom `extract_jinja` function to pick up the `OPTIONS` section. Without this, the `jinja2` extractions will not work well at all.